e2e/helm/templates/kafka.yaml.gotmpl

```
@@ -0,0 +1,30 @@
+cp-zookeeper:
+  imageTag: {{ .Values | get "zookeeper.version" "7.7.1" }}
```

I noticed we end up running zookeeper kafka version `7.7.2` in e2e. Should we update these defaults to be the same?
```suggestion
  imageTag: {{ .Values | get "zookeeper.version" "7.7.2" }}
```